apps/frontend/src/modules/users/users-api.ts

```diff
@@ -4,10 +4,7 @@ import { type HTTP } from "~/libs/modules/http/http.js";
 import { type Storage } from "~/libs/modules/storage/storage.js";
 
 import { UsersApiPath } from "./libs/enums/enums.js";
-import {
-	type UserGetAllItemResponseDto,
-	type UserGetAllResponseDto,
-} from "./libs/types/types.js";
+import { type UserGetAllItemResponseDto } from "./libs/types/types.js";
```

Comment:
```suggestion
import { type UserDto } from "./libs/types/types.js";
```

Reply:
Should I update on any other file that this import exists or just this one?

Reply:
I think you can completely remove that type from web side. Update in other places as well